src/ui/state/timeline.ts

```diff
@@ -25,15 +20,12 @@ export interface TimelineState {
     stalled?: boolean;
   } | null;
   playbackPrecachedTime: number;
-  screenShot: ScreenShot | null;
-  mouse: MouseAndClickPosition | null | undefined;
   recordingDuration: number | null;
   zoomRegion: ZoomRegion;
   timelineDimensions: { width: number; left: number; top: number };
   hoverTime: number | null;
   unprocessedRegions: TimeRange[];
   shouldAnimate: boolean;
-  tooltip: Tooltip | null;
```

Comment:
Nice, awesome to see this! I feel like CSS and redux are the two places where it is easiest to just leave unused code forever.